### ISSUE TITLE
Round/Digits not working as documented?

### ISSUE THREAD

**Issue by perfguy** [Rapporter/pander#285]
I want to print the following with exactly two digits after the decimal point. I am not able to accomplish that with round and/or keep.trailing.zeros

```{r}
> r <- matrix(c(283764.97430, 29.12345678901, -7.1234, -100.10), ncol = 2)
> pandoc.table(r, keep.trailing.zeros=T, round=2)

------ ------
**283765** -7.12 

29.12  **-100.1**
------ ------
```

**Comment by daroczig** [comment]
I think this was fixed in #259, can you please give a try to the dev version?

**Comment by perfguy** [comment]
yes this is fixed in the dev version. Thanks!
